src/app/routes/edges/routes/PrivateEdgeList/index.js

```diff
@@ -1349,7 +1351,10 @@ class PrivateEdgeList extends React.Component {
                                   iconOnly={true}
                                 />
                               )}
-                              {this.state.userRole !== "READ_ONLY_OPS" && (
+                              {!(
+                                roles.includes("NAMESPACE_READ_ONLY") ||
```

Comment:
let's handle based on aggregated permissions and not roles so custom roles are included.

cluster.write permission gives access to manage clusters.

also consider project specific scenarios